src/asm/symbol.cpp

```diff
@@ -90,40 +89,39 @@ std::string *Symbol::getEqus() const {
 }
 
 static void dumpFilename(Symbol const &sym) {
-	if (sym.src)
+	if (sym.src) {
 		sym.src->dump(sym.fileLine);
-	else if (sym.fileLine == 0)
-		fputs("<command-line>", stderr);
-	else
-		fputs("<builtin>", stderr);
-}
-
-// Set a symbol's definition filename and line
-static void setSymbolFilename(Symbol &sym) {
```

Comment:
Why remove this function?

Reply:
It's an extra level of indirection that was making it that little bit harder to see how the fstk `referenced` and `ID` values are propagating. Also it was only a two-line function that's called three times, and one of those three calls was subsequently overwriting one of the two lines (macros set their own `fileLine`).